tests/integration/components/post-contribution-form/component-test.js

```diff
@@ -58,9 +59,12 @@ describe('Integration | Component | post contribution form', function() {
     this.set('model', {
       user: {},
       title: 'My Contr',
-      date: '2016-11-21',
+      date: moment().format('YYYY-MM-DD'),
```

Comment:
Hmm, they shouldn't fail, as this always uses "today" and "yesterday", or am I making a fallacy here? I will still adjust them, as I find it more clear, explicit and safer